### ISSUE TITLE
Easy selection of a learning rate scheduler when initializing a Trainer

### ISSUE THREAD

**Issue by freeIsa** [huggingface/transformers#5936]
# 🚀 Feature request

Please consider adding a new argument to the Trainer constructor to specify which among the available learning rate schedulers should be used during training.

## Motivation

Even though Trainer already has the option to specify a given optimizer and learning rate scheduler, you need to explicitly initialize both (even when you only want to change the scheduler) with parameters already available to the Trainer itself via TrainingArguments. It would be nicer and smoother to just provide Trainer with a string specifying which scheduler to use (e.g. 'constant_schedule', 'cosine_schedule_with_warmup', ...) and have `get_optimizers` implement the choice.
